modules/geo-layers/src/terrain-layer/terrain-layer.js

```diff
@@ -110,6 +110,7 @@ export default class TerrainLayer extends CompositeLayer {
     loadOptions = {
       ...loadOptions,
       terrain: {
+        skirtHeight: this.state.isTiled ? meshMaxError * 2 : 0,
         ...loadOptions?.terrain,
```

Comment:
Setting `skirtHeight` before `...loadOptions` means a value in `loadOptions` will overwrite the default value of `skirtHeight`, right?

Reply:
I think this does what you're hoping it does. Providing a skirtHeight in the loader options will override this default, but providing other options will not override default skirtHeight accidentally.

Reply:
Yes, I wanted a default that would be overridden through `loadOptions`, but I always have to double check whether `{...a, ...b}` has `a` overwriting `b` or vice versa 😄